src/web.ts

```diff
@@ -1,22 +1,61 @@
-import { WebPlugin } from '@capacitor/core';
-import { SignInWithApplePlugin } from './definitions';
+import { WebPlugin } from "@capacitor/core";
 
-export class SignInWithAppleWeb extends WebPlugin implements SignInWithApplePlugin {
+import { SignInWithApplePlugin, InitOptions } from "./definitions";
+
+declare var window: any;
+
+export class SignInWithAppleWeb extends WebPlugin
+  implements SignInWithApplePlugin {
   constructor() {
     super({
-      name: 'SignInWithApple',
-      platforms: ['web']
+      name: "SignInWithApple",
+      platforms: ["web"],
     });
   }
 
-  async Authorize(): Promise<{response: any}> {
+  async SignIn(): Promise<void> {
```

Comment:
I don't really see the need for this method - you're doing exactly the same work in the `Authorize()` method (which also matches the iOS Plugin API) - just remove this and stick with the other one?

Reply:
You are correct, addressed in db36e4f4f5497465dc393012e993a4e86fce8b50